timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -413,16 +420,18 @@ TIBDExchangeMove<RealType>::move_host(const int N, const double *h_coords, const
 template <typename RealType> double TIBDExchangeMove<RealType>::raw_log_probability_host() {
     std::vector<RealType> h_log_exp_src(2 * this->batch_size_);
     std::vector<RealType> h_log_exp_after(2 * this->batch_size_);
+
     d_lse_max_src_.copy_to(&h_log_exp_src[0]);
     d_lse_exp_sum_src_.copy_to(&h_log_exp_src[this->batch_size_]);
+
     this->d_lse_max_after_.copy_to(&h_log_exp_after[0]);
     this->d_lse_exp_sum_after_.copy_to(&h_log_exp_after[this->batch_size_]);
 
     int h_targeting_inner_vol[this->batch_size_];
     d_targeting_inner_vol_.copy_to(h_targeting_inner_vol);
 
-    int h_local_inner_count[1];
-    d_inner_mols_count_.copy_to(h_local_inner_count);
+    int h_local_inner_count;
+    d_inner_mols_count_.copy_to(&h_local_inner_count);
```

Comment:
Was this the bugfix?

Reply:
OK, thanks. Now I see that before and after are equivalent also considering the change on the line above.